client/src/components/History/GanttVisualisation/Gantt.vue

```diff
@@ -1,13 +1,48 @@
 <template>
     <div>
         <svg id="gantt"></svg>
-        <button id="view" @click="changeQDayView">Quarter Day View</button>
-        <button id="view" @click="changeHDayView">Half Day View</button>
-        <button id="view" @click="changeDayView">Day View</button>
-        <button id="view" @click="changeWeekView">Week View</button>
-        <button id="view" @click="changeMonthView">Month View</button>
-        <button id="view" @click="changeHourView">Hour View</button>
-        <button id="view" @click="changeMinuteView">Minute View</button>
+        <div class="sticky">
```

Comment:
Looks Good, can we please make this Modal as a separate Vue-composable? it will just be cleaner that way, what say?